platform/measurement_utils.cpp

```diff
@@ -50,15 +45,15 @@ string ToStringPrecisionLocale(Locale loc, double d, int pr)
   {
     // Value with no decimals. Check if it's equal or bigger than 10000 to
     // insert the grouping (thousands) separator characters.
-    if (out.size() > 4 && !loc.m_groupingSeparator.empty())
```

Comment:
This check `if (out.size() > 4 &&` is needed to add the grouping separators to values equal or bigger than 10000. If we remove it, then also values from 1000 to 9999 will have the grouping separator.

Do we really want this? In the past it was decided to add the grouping separators with values >= 10000.

Reply:
Thanks for reminding about that, I've reverted the change. Do you remember that reasoning? :)

Reply:
Here's the reasoning that we followed last year for the grouping separator: https://github.com/organicmaps/organicmaps/pull/4029#discussion_r1297801741